ui/src/app/utils/runtime-hooks.tsx

```diff
@@ -147,6 +147,11 @@ export const useDisk = (currentWorkspaceNamespace: string) => {
   }, [currentWorkspaceNamespace]);
 };
 
+let aborter = new AbortController();
+
+// This function is being used by test ONLY to abort the runtime polling so that the tests do not interfere with each
+// other
+export const getAborter = () => aborter;
 // useRuntimeStatus hook can be used to change the status of the runtime
```

Comment:
add a new line here, so it's clear that this is not related to the aborter

Reply:
Done